components/brave_today/browser/publishers_parsing.cc

```diff
@@ -36,7 +36,15 @@ bool ParseCombinedPublisherList(const std::string& json,
     publisher->publisher_id = *publisher_raw.FindStringKey("publisher_id");
     publisher->type = mojom::PublisherType::COMBINED_SOURCE;
     publisher->publisher_name = *publisher_raw.FindStringKey("publisher_name");
+
     publisher->category_name = *publisher_raw.FindStringKey("category");
+    auto* channels_raw = publisher_raw.FindListKey("channels");
```

Comment:
FYI, `FindListKey` is deprecated. It would be good to use `Dict::FindList()` in the new code.

Reply:
Good call! I've migrated the `FindListKey` stuff over. Presumably the rest should be updated too, but I'd like to do that in a follow up PR.